models/Recipe.js

```diff
@@ -1,8 +1,18 @@
 const mongoose = require('mongoose');
 const Schema = mongoose.Schema;
 
+
 const recipeSchema = new Schema({
-  // TODO: write the schema
+  title: {type :String},
+  level : {type :String},
+  ingredients : {type:Array},
+  cuisine: {type:String},
+  dishType: {type:String},
+  image: {type:String,default:'https://images.media-allrecipes.com/images/75131.jpg'},
+  duration : {type :Number, min:0},
+  creator: {type:String},
+  created : {type: String,default : new Date() }
+
 });
 
```

Comment:
Regarding the Schema, you were asked to be more precise, so when you enter (or more importantly, a user enter) a new recipe, you are sure that they are formated the good way 
`
const recipeSchema = new Schema({
  title: { type: String, required: true, unique: true },
  level: {
    type: String,
    enum: ["Easy Peasy", "Amateur Chef", "UltraPro Chef"]
  },
  ingredients: [],
  cuisine: { type: String, required: true },
  dishType: {
    type: String,
    enum: ["Breakfast", "Dish", "Snack", "Drink", "Dessert", "Other"]
  },
  image: {
    type: String,
    default: "https://images.media-allrecipes.com/images/75131.jpg"
  },
  duration: { type: String, min: 0 },
  creator: String,
  created: { type: Date, default: Date.now }
});

`